### DISCUSSION TITLE
Text markers on images

### DISCUSSION THREAD

**Discussion by jtbandes** [foxglove/community#611] [Q&A]
Image annotations don't currently support text, although Foxglove Studio has some code to support them if the user brings a self-modified message type. We should support and document this through our image annotations/markers.

**Comment by amacneil** [comment]
Related feature request - would be awesome to support rendering "icon" markers (image + 3d), like webviz demo mode supports. We could either ship with some standard icons, or let you reference a URL or something.

**Comment by jtbandes** [comment]
File a separate ticket?
